pkg/types/powervs/powervs_regions.go

```diff
@@ -38,6 +38,11 @@ var Regions = map[string]Region{
 			"lon06",
 		},
 	},
+	"mon": {
+		Description: "Montreal, Canada",
+		VPCRegion:   "ca-tor",
```

Comment:
i remember there being a `ca-mon` region but we don't push the rhcos vhds to that region. is `ca-tor` the correct vpc region ?

Reply:
We are only aware of `ca-tor` being available in IBM Cloud. I just took a quick look at the list of regions available and still do not see ca-mon being an option. The intent is to choose a data center that is close to Montreal as the default for mon01.